### PR TITLE
Add Dutch sentences to set lights

### PR DESCRIPTION
**Pull request by TheFes**

### CODE REVIEW

File: sentences/nl/light_HassLightSet.yaml [hunk 1]
```diff
@@ -1,4 +1,18 @@
 language: nl
 intents:
   HassLightSet:
-    data: []
+    data:
+      # brightness
+      - sentences:
+          - "zet <name> op <brigtness> procent"
+          - "zet [de] helderheid van <name> op <brightness> [procent]"
```
Comment (frenck) [inline sentences/nl/light_HassLightSet.yaml:8]: Maybe add `felheid`, `itensiteit` too?

File: sentences/nl/light_HassLightSet.yaml [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+          - "verander [de] helderheid van <name> naar <brightness> [procent]"
+      - sentences:
+          - "zet [de] helderheid in [de][het] <area> op <brightness> [procent]"
+          - "zet <area> helderheid op <brightness> [procent]"
+        slots:
+          name: "all"
+      # color
+      - sentences:
+          - "zet <name> op {color}"
```
Comment (frenck) [inline sentences/nl/light_HassLightSet.yaml:17]: maybe add:
- `maak <name> {color}` ?

File: sentences/nl/light_HassLightSet.yaml (hunk 1 above)
Comment (TheFes) [reply sentences/nl/light_HassLightSet.yaml:8]: Good one, will do this evening

File: sentences/nl/light_HassLightSet.yaml [hunk 3]
```diff
@@ -1,4 +1,23 @@
 language: nl
 intents:
   HassLightSet:
-    data: []
+    data:
+      # brightness
+      - sentences:
+          - "zet <name> op <brigtness> procent"
```
Comment (balloob) [inline sentences/nl/light_HassLightSet.yaml:7]: You are referring to expansion rule `<brigtness>` (typo btw, missing h).

This expansion rule is not defined in `sentences/nl/_common.yaml`. Take a look at how English does it:

 - specify a list to define valid values of brightness (0…100)
 - specify an expansion rule that catches a list item, then renames it to `brightness_pct`, which is the valid slot name for the HassLightSet intent

https://github.com/home-assistant/intents/blob/9004373ef6350f4829a30c8ecfeb0e14da9057b3/intents.yaml#L61-L63

File: sentences/nl/light_HassLightSet.yaml [hunk 4]
```diff
@@ -1,4 +1,23 @@
...
+      - sentences:
+          - "zet [de] helderheid in [de][het] <area> op <brightness> [procent]"
+          - "zet <area> helderheid op <brightness> [procent]"
+        slots:
+          name: "all"
+      # color
+      - sentences:
+          - "zet <name> op {color}"
+          - "verander [[de] kleur van] <name> naar {color}"
+          - "maak <name> {color}"
```
Comment (balloob) [inline sentences/nl/light_HassLightSet.yaml:23]: Colors also need to be defined in the common file. I would limit this PR to just brightness, do color once it's merged.

File: sentences/nl/light_HassLightSet.yaml (hunk 3 above)
Comment (TheFes) [reply sentences/nl/light_HassLightSet.yaml:7]: Ah, that was the missing link :) I will update it

File: sentences/nl/_common.yaml [hunk 5]
```diff
@@ -7,12 +7,20 @@ responses:
...
+      from: 0
+      to: 100
 expansion_rules:
   name: "[de | het] {name}"
   area: "[de | het] {area}"
-  doe_enkelvoud: "(zet | mag | doe)"
-  doe_meervoud: "(zet | mogen | doe)"
+  doe_enkelvoud: "(zet | mag | doe | verander)"
+  doe_meervoud: "(zet | mogen | doe | verander)"
+  doe_naar: "(op | naar)"
```
Comment (balloob) [inline sentences/nl/_common.yaml:21]: hoezo zit `doe` in de naam van deze expansion rule ?

**Review by balloob — APPROVED** [review]

**Pull request merged**